CONTRIBUTING.md

```diff
@@ -104,6 +104,30 @@ a ticket number.
 * After feedback has been given we expect responses within two weeks. After two
   weeks we may close the pull request if it isn't showing any activity.
 
+## Revert Policy
+By running tests in advance and by engaging with peer review for prospective
+changes, your contributions have a high probability of becoming long lived
+parts of the the project. After being merged, the code will run through a
+series of testing pipelines on a large number of operating system
+environments. These pipelines can reveal incompatibilities that are difficult
+to detect in advance.
+
+If the code change results in a test failure, we will make our best effort to
```

Comment:
@johnduarte I like this amended change - it makes more sense to me (thinking globally) to consider the merge => test failure => revert as a static time window rather than adjusting based on relation to a specific Portland time.